test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -1,230 +1,732 @@
+            $newFile = Move-Item -Path $testFile -Destination "moveFile.txt" -PassThru
+            $fileExists = Test-Path "moveFile.txt"
+            $fileExists | Should Be $true
+            $newFile.Name | Should Be "moveFile.txt"
+        }
+
+        It "Verify Get-ChildItem" {
+            $dirContents = Get-ChildItem "."
+            $dirContents.Count | Should Be 2
+        }
     }
 
-    It "Verify Remove-Item for file" {
-        $existsBefore = Test-Path $testFile
-        Remove-Item -Path $testFile -Force
-        $existsAfter = Test-Path $testFile
-        $existsBefore | Should Be $true
-        $existsAfter | Should Be $false
+    Context "Validate basic host navigation functionality" {
+        BeforeAll {
+            #build semi-complex directory structure to test navigation within
+            $level1_0 = "Level1_0"
+            $level2_0 = "Level2_0"
+            $level2_1 = "Level2_1"
+            $root = Join-Path "TestDrive:" "" #adds correct / or \
+            $level1_0Full = Join-Path $root $level1_0
+            $level2_0Full = Join-Path $level1_0Full $level2_0
+            $level2_1Full = Join-Path $level1_0Full $level2_1
+            New-Item -Path $level1_0Full -ItemType Directory > $null
+            New-Item -Path $level2_0Full -ItemType Directory > $null
+            New-Item -Path $level2_1Full -ItemType Directory > $null
+        }
+
+        It "Verify Get-Location and Set-Location" {
+            $currentLoc = Get-Location
+            Set-Location $level1_0
+            $level1Loc = Get-Location
+            Set-Location $level2_0
+            $level2Loc = Get-Location
+            $currentLoc.Path | Should Be $root
+            $level1Loc.Path | Should Be $level1_0Full
+            $level2Loc.Path | Should Be $level2_0Full
+        }
+
+        It "Verify Push-Location and Pop-Location" {
+            #push a bunch of locations
+            Push-Location
+            $push0 = Get-Location
+            Set-Location $level1_0
+            Push-Location
+            $push1 = Get-Location
+            Set-Location $level2_0
+            Push-Location
+            $push2 = Get-Location
+
+            #navigate back home to change path out of all pushed locations
+            Set-Location "TestDrive:\"
+
+            #pop locations off
+            Pop-Location
+            $pop0 = Get-Location
+            Pop-Location
+            $pop1 = Get-Location
+            Pop-Location
+            $pop2 = Get-Location
+
+            $pop0.Path | Should Be $push2.Path
+            $pop1.Path | Should Be $push1.Path
+            $pop2.Path | Should Be $push0.Path
+        }
     }
 
-    It "Verify Copy-Item" {
-        $newFile = Copy-Item -Path $testFile -Destination "copyFile.txt" -PassThru
-        $fileExists = Test-Path "copyFile.txt"
-        $fileExists | Should Be $true
-        $newFile.Name | Should Be "copyFile.txt"
+    Context "Validate basic Content Cmdlets for the FileSystem provider" {
```

Comment:
You are right. I merged both those contexts into one under Context "Validate basic FileSystem Cmdlets"